go.mod

```diff
@@ -43,11 +43,11 @@ require (
 	github.com/hashi-derek/grpc-proxy v0.0.0-20231207191910-191266484d75
 	github.com/hashicorp/consul-awsauth v0.0.0-20220713182709-05ac1c5c2706
 	github.com/hashicorp/consul-net-rpc v0.0.0-20221205195236-156cfab66a69
-	github.com/hashicorp/consul/api v1.29.4
-	github.com/hashicorp/consul/envoyextensions v0.7.3
+	github.com/hashicorp/consul/api v1.31.0
+	github.com/hashicorp/consul/envoyextensions v0.7.7
 	github.com/hashicorp/consul/proto-public v0.6.2
```

Comment:
I want to learn, why we skipped the proto-public update? I can see the latest patch version is v0.6.3